netty/shaded/native-image.properties

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 The Netty Project
+#
+# The Netty Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+Args = --initialize-at-build-time=io.grpc.netty.shaded.io.netty \
```

Comment:
We don't want this hard-coded in gRPC. If you really wanted to avoid making the transformer/relocator, then you could extract the file from the Netty jar dependency. But hard-coding it here is a dead-end.